redaxo/src/addons/structure/plugins/content/boot.php

```diff
@@ -10,6 +10,7 @@
 
 rex_perm::register('moveSlice[]', null, rex_perm::OPTIONS);
 rex_complex_perm::register('modules', 'rex_module_perm');
+rex_complex_perm::register('functions', 'rex_functions_perm');
```

Comment:
Für meinen Geschmack ist der Begriff "functions" alleine zu Allgemein.. vllt besser "articleFunctions" oder so?

@gharlan ggf noch ne andere idee?

Reply:
Würde ich hier nicht angehen. Vielleicht kommen wir in Zukunft komplett an dem Reiter vorbei

Reply:
D.h. Du würdest den PR komplett ablehnen?

Reply:
Da es sich nur um eine Automatik handelt, würde es für den User in den Einstellungen nichts ändern. Daher sehe ich es als unproblematisch, die Funktionalität einfach zu übernehmen. Für die Zukunft würde ich es sowieso komplett umbauen (siehe Slack)

Reply:
Ich würde die Complex-Perm auch anders nennen. Zum Beispiel `structure_functions`?